src/main/webapp/creatives/maps/simid_map_creative.test.js

```diff
@@ -119,9 +143,9 @@ test('testing that rejection for ad params not found is working', () => {
         type: "SIMID:Player:init",
     };
     testMap.onInit(eventData);
-    
-    const instance = SimidProtocol.mock.instances[0];
-    const rejectMessageObject = instance.reject.mock.calls[0][1];
+
```

Comment:
Just a general question but have you looked into using the createInitData function I added in here? It might save you some time from having to repeat the eventData declaration in every test

Reply:
thanks for pointing that out! Refactored tests to use that function